tests/node_compat/test/parallel/test-sqlite-statement-sync.js

```diff
@@ -0,0 +1,326 @@
+// deno-fmt-ignore-file
+// deno-lint-ignore-file
+
+// Copyright Joyent and Node contributors. All rights reserved. MIT license.
+// Taken from Node 23.9.0
+// This file is automatically generated by `tests/node_compat/runner/setup.ts`. Do not modify this file manually.
+
+'use strict';
+require('../common');
+const tmpdir = require('../common/tmpdir');
+const { join } = require('node:path');
+const { DatabaseSync, StatementSync } = require('node:sqlite');
+const { suite, test } = require('node:test');
+let cnt = 0;
+
+tmpdir.refresh();
+
+function nextDb() {
+  return join(tmpdir.path, `database-${cnt++}.db`);
+}
+
+suite('StatementSync() constructor', () => {
+  test('StatementSync cannot be constructed directly', (t) => {
+    t.assert.throws(() => {
+      new StatementSync();
+    }, {
+      code: 'ERR_ILLEGAL_CONSTRUCTOR',
+      message: /Illegal constructor/,
+    });
+  });
+});
+
+suite('StatementSync.prototype.get()', () => {
+  test('executes a query and returns undefined on no results', (t) => {
+    const db = new DatabaseSync(nextDb());
+    t.after(() => { db.close(); });
+    let stmt = db.prepare('CREATE TABLE storage(key TEXT, val TEXT)');
```

Comment:
Just to be 100% sure - have you verified that these actually pass and are not silently ignored by `deno test` is something goes wrong? If so, our support for `node:test` is getting pretty good 💪

Reply:
yep, i was painfully going through fixing each test as the previous one passed so it does actually work!